.github/workflows/stale_prs.yml

```diff
@@ -0,0 +1,19 @@
+name: Mark pull requests as stale
```

Comment:
dupe?

Reply:
Noted and read file name vs the other one, they had it separated, you saw what I said in discord 🤷